sql/select.go

```diff
@@ -285,50 +286,17 @@ func (v *indexInfo) analyzeRanges(exprs []parser.Exprs) {
 // if it matches the ordering requested by the query. Non-matching orderings
 // increase the cost of using the index.
 func (v *indexInfo) analyzeOrdering(scan *scanNode, ordering []int) {
-	// Compute the ordering provided by the index.
-	indexOrdering := scan.computeOrdering(v.index.fullColumnIDs())
-
 	// Compute the prefix of the index for which we have exact constraints. This
 	// prefix is inconsequential for ordering because the values are identical.
-	prefix := exactPrefix(v.constraints)
+	v.exactPrefix = exactPrefix(v.constraints)
+
+	// Compute the ordering provided by the index.
+	indexOrdering := scan.computeOrdering(v.index.fullColumnIDs())
 
 	// Compute how much of the index ordering matches the requested ordering for
 	// both forward and reverse scans.
-	fwdMatch, fwdPrefix, fwdIndexOrdering := 0, prefix, indexOrdering
```

Comment:
Not quite. The `computeOrderingMatch` this was refactored into is also used by `sortNode.wrap`.